File: lib/pusher.ts
import Pusher from "pusher"

if (!process.env.PUSHER_SECRET || !process.env.PUSHER_APP_ID || !process.env.PUSHER_KEY || !process.env.PUSHER_CLUSTER) {
    throw Error('PUSHER env has not been set')
}

export const pusher = new Pusher({
    appId: process.env.PUSHER_APP_ID,
    key: process.env.PUSHER_KEY,
    secret: process.env.PUSHER_SECRET,
    cluster: process.env.PUSHER_CLUSTER
})

File: client/pusher.py
import os
import pysher
from dotenv import load_dotenv

load_dotenv()

PUSHER_CLUSTER = os.getenv("PUSHER_CLUSTER")
PUSHER_KEY = os.getenv("PUSHER_KEY")

if not PUSHER_CLUSTER or not PUSHER_KEY:
    print("Please provide PUSHER_SECRET, PUSHER_CLUSTER, PUSHER_APP_ID, and PUSHER_KEY in .env file")
    exit(1)


pusher = pysher.Pusher(PUSHER_KEY, PUSHER_CLUSTER)